guide/additional-info/hosting.md

```diff
@@ -0,0 +1,68 @@
+# Hosting your bot on a VPS
+
+A bot requires itself to be hosted in order for it to run, however, it would not be ideal to host it on your computer all day without shutting it down. That's where a **Virtual Private Server** (VPS) comes into place. A good VPS can mark a turning point for your project as you can host local databases, and it is reliable as well. A VPS suitable for your Discord bot is typically cheap, ranging from around $2-10/month.
+
+## Well known affordable providers
+
+We have gathered a small list of reputable and inexpensive VPS providers that you may rely on for getting your job done:
+* [OVH](https://www.ovh.com/us/vps/) - Starting at $3.35/month for 1 core, 2GB RAM, 20GB SSD
+* [DigitalOcean](https://www.digitalocean.com/) - Starting at $5/month for 1 core, 1GB RAM, 25GB SSD
+* [Linode](https://www.linode.com/) - Starting at $5/month for 1 core, 1GB RAM, 25GB SSD
+* [Vultr](https://www.vultr.com/) - Starting at $2.50/month for 1 core, 512MB RAM, 10GB SSD
+* [AWS Lightsail](https://amazonlightsail.com/) - Starting at $3.50/month (first month free) for 1 core, 512MB RAM, 20GB SSD
+
+Before you purchase a VPS, we recommend learning how to remotely connect to a VPS. The usual method is via [SSH](https://en.wikipedia.org/wiki/Secure_Shell_Protocol). We won't go deep into it as it's outside the scope of this guide, however, VPS providers usually have a guide regarding connection so you can read up on that.
+
+## Steps to host
+
+::: warning
+The following steps stated assumes your Linux distribution is Ubuntu. However, all Linux distributions are similar and should work on your VPS.
+:::
+
+Presuming that now you have purchased your own VPS and that you have remotely connected to it, what should you do now? We recommend you create a Github repository first and add your bot files there as it would be very inefficient to code files through a terminal.
```

Comment:
```suggestion
Once you have purchased your VPS and remotely connected to it, what should you do now? We recommend you create a GitHub repository first and add your bot files there, as it would be very inefficient to code files through a terminal.
```

Reply:
Will need to remove the Git Wikipedia page link below if this suggestion is applied
```suggestion
Once you have purchased a VPS and are remotely connected to it, the next step is to clone the bot's source code onto the server's file system. We recommend you store your bot's source code in a [GitHub](https://github.com) repository and use [Git](https://git-scm.com) to clone the files.
```

Reply:
Wouldnt need to add those extra hyper-links @lavgup.

Reply:
Why not?

Reply:
> Wouldnt need to add those extra hyper-links @lavgup. 

Sorry, saw this comment but forgot to leave a reply. I would think that these hyperlinks would be a necessary addition to the page as a convenient way for users to navigate to the linked website faster. They're the first time the products are mentioned in the page; they aren't "extra".


Also, hyperlinks aside, I believe the rewording would be beneficial overall as it is easier to read, more formal and less blog-like. The user knows exactly what they're going to do next and exactly what they will be using; it gets straight to the point.